apps/publisher/src/app.tsx

```diff
@@ -242,26 +240,26 @@ function App() {
     <ControlBar
       controls={[
         {
+          icon: isAudioEnabled ? <IconMicrophoneOn /> : <IconMicrophoneOff />,
+          isActive: !isAudioEnabled,
+          isDisabled: !(mediaStream && mediaStream.getAudioTracks().length),
           key: 'toggleMicrophoneButton',
-          'test-id': 'toggleMicrophoneButton',
-          tooltip: { label: 'Toggle microphone', placement: 'top' },
           onClick: () => {
             toggleAudio();
           },
-          isActive: !isAudioEnabled,
-          isDisabled: !(mediaStream && mediaStream.getAudioTracks().length),
-          icon: isAudioEnabled ? <IconMicrophoneOn /> : <IconMicrophoneOff />,
+          testId: 'toggleMicrophoneButton',
+          tooltipProps: { label: 'Toggle microphone', placement: 'top' },
         },
         {
+          icon: isVideoEnabled ? <IconCameraOn /> : <IconCameraOff />,
+          isActive: !isVideoEnabled,
+          isDisabled: !(mediaStream && mediaStream.getVideoTracks().length),
           key: 'toggleCameraButton',
-          'test-id': 'toggleCameraButton',
-          tooltip: { label: 'Toggle camera', placement: 'top' },
           onClick: () => {
             toggleVideo();
           },
-          isActive: !isVideoEnabled,
-          isDisabled: !(mediaStream && mediaStream.getVideoTracks().length),
-          icon: isVideoEnabled ? <IconCameraOn /> : <IconCameraOff />,
+          testId: 'toggleCameraButton',
+          tooltipProps: { label: 'Toggle camera', placement: 'top' },
```

Comment:
updated props and reordered alphabetically:
`'test-id'` -> `testId`
`tooltip` -> `tooltipProps`